content/kapp-controller/docs/latest/air-gapped-workflow.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Air-gapped Workflow
+---
+
+The documentation below covers topics from the [imgpkg air-gapped workflow docs](/imgpkg/docs/latest/air-gapped-workflow) 
+more concisely in order to focus on applying these workflows to kapp-controller PackageRepositories. 
+
+## Scenario
+
+You have a [PackageRepository](packaging#packagerepository-cr) bundle or image in an external OCI registry that you would like 
+to move into an OCI registry in an air-gapped environment. Once relocated, you would like to deploy the bundle or image as part 
+of a PackageRepository to a Kubernetes cluster.
+
+## Prerequisites
+
+In order to go through this process of moving an [imgpkg bundle](/imgpkg/docs/latest/resources/#bundle) or image to an 
+air-gapped environment, you will need to have [imgpkg](/imgpkg) installed. More information on installing Carvel tools, 
+including `imgpkg` can be found [here](/#whole-suite).
+
+## PackageRepository Format
+
+When going through this air-gapped scenario for kapp-controller PackageRepositories, you will most commonly be relocating a bundle of bundles 
+(i.e. a recursive bundle) from an external OCI registry to an OCI registry in your air-gapped environment. 
+
+When working with kapp-controller PackageRepositories, it is recommended to use the [PackageRepository bundle format](packaging#package-repository-bundle-format). 
+In this format, a PackageRepository is represented as an imgpkg bundle. It is also recommended that Packages that are part of a PackageRepository be imgpkg bundles 
+as documented [here](packaging#package-bundle-format). 
```

Comment:
this content here seems to be describing what package repo authors need to know, not consumers of repos. even if consumers read package repo docs it's not like they can change what they were given by authors.

Reply:
My original thought here was it made sense to discuss what a PackageRepo consumer would be relocating in this case, but I think it makes sense as you say to not get into these details as it is covered in external docs that are linked.